src/main/java/httpserver/http/response/Response.java

```diff
@@ -0,0 +1,39 @@
+package httpserver.http.response;
+
+import httpserver.http.Protocol;
+import httpserver.http.StatusCode;
+
+public class Response {
+    private Protocol protocol;
+    private StatusCode statusCode;
+    private String headers;
+
+    public Response setProtocol(Protocol protocol) {
+        this.protocol = protocol;
+        return this;
+    }
+
+    public Response setStatusCode(StatusCode statusCode) {
+        this.statusCode = statusCode;
+        return this;
+    }
+
+    public Response setHeaders(String headers) {
+        this.headers = headers;
+        return this;
+    }
+
+    public String build() {
```

Comment:
Think this can be taken one stage further, if there is a `ResponseBuilder` that has a `build` method that returns a `Response` object. The `Response` object can then have a `toString` method which does what this `build` method does.

Reply:
Thank you, Matt. I'll change this